src/package/nuspec/Microsoft.TestPlatform.Portable.nuspec

```diff
@@ -96,6 +97,26 @@
     <file src="net451\$Runtime$\testhost.net472.x86.exe.config"	target="tools\net451\testhost.net472.x86.exe.config" />
     <file src="net451\$Runtime$\testhost.net48.x86.exe"	target="tools\net451\testhost.net48.x86.exe" />
     <file src="net451\$Runtime$\testhost.net48.x86.exe.config"	target="tools\net451\testhost.net48.x86.exe.config" />
+
+    <file src="net451\$Runtime$\testhost.arm64.exe"	target="tools\net451\testhost.arm64.exe" />
```

Comment:
Funny to see we still have tabs in this file. I probably did something wrong in the `.editorconfig`.

Reply:
I reformatted the file, at least there are no huge changes apart from my changes.